README.md

```diff
@@ -165,11 +165,30 @@ jobs:
       - uses: erlef/setup-beam@v1
         with:
           otp-version: '24'
-          gleam-version: '0.16'
-          rebar3-version: '3.16'
-      - run: rebar3 ct
+          gleam-version: '0.19.0-rc3'
+      - run: gleam test
+```
+
+### Example (Gleam on Ubuntu without OTP)
+
```

Comment:
```suggestion
Note that `otp: false` currently is only applicable to gleam. 
```

Reply:
@paulo-ferraz-oliveira Thoughts?

Reply:
I included this note below the example snippet. Should it be moved up?

Reply:
Ahh, nah that's good. I just didn't see, it should read fine in the actual README vs a PR :)